casper/blockchain/blockchain_protocol.py

```diff
@@ -3,13 +3,16 @@
 from casper.blockchain.blockchain_plot_tool import BlockchainPlotTool
 from casper.protocol import Protocol
 
+
 class BlockchainProtocol(Protocol):
     View = BlockchainView
     Message = Block
     PlotTool = BlockchainPlotTool
 
-    genesis_block = Block(None, dict(), None, 0, -1)
+    genesis_block = None
 
-    @staticmethod
-    def initial_message(validator):
-        return BlockchainProtocol.genesis_block
+    @classmethod
```

Comment:
What's the difference between a classmethod and staticmethod here?

Reply:
The static_method is assuming we aren't going to mess around with the class or a specific object. Because I set stuff in the class, I want to be clear that I am doing that. Using staticmethod and then bringing in the class explicitly to change one of the class vars is a bit dishonest in our intentions.

staticmethod was fine just as a getter as it was previously, but I think classmethod is better here.